packages/animations/example/lib/container_transition.dart

```diff
@@ -117,18 +122,63 @@ class _OpenContainerTransformDemoState
       body: ListView(
         padding: const EdgeInsets.all(8.0),
         children: <Widget>[
+          Container(
```

Comment:
It would probably be sufficient to just show a snackbar for all items instead of adding this to the scroll view.